js/alerts.js

```diff
@@ -196,6 +197,143 @@ function updateStatus(alertid, duplicate, bugid, mergedfrom) {
     }
 }
 
+
+//function to check whether a name exists in an array
+function containsObject(obj, list) {
+    var i;
+    for (i = 0; i < list.length; i++) {
+        if (list[i] === obj) {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+function performAction(){
+    var action = $(document.getElementById("actions")).val();
+    console.log(action);
+
+    var status_options = ["NEW", "Back Filling", "Investigating", "Duplicate", "Resolved", "Shipped", "Wont Fix", "False Alarm", "Ignore", "Not Tracking", "Backout", "Too Low"];
+
+    //get ids of all the checked alerts
+   var checkedIds = $(":checkbox:checked").map(function() {
+        return this.id;
+    }).get();
+   //Check if atleast one alert is chosen
+   if (checkedIds.length<=0 && action != "Actions"){
+    console.log("No alert chosen");
+    alert("Please Choose atleast one Alert");
+   }
+   else{
+
+        for (alert in checkedIds){
+            checkedIds[alert] = checkedIds[alert].split('-')[1];
+        }
+        console.log(checkedIds);
+        //Check if status has to be changed
+        if (containsObject(action, status_options)){
+            console.log("change status-"+action);
+            //Update the status one by one by calling updatestatus()
+            for (id in checkedIds){
+                $.ajax({
+                     url: root_url + "/updatestatus",
+                    type: "POST",
+                    data: {
+                        id: checkedIds[id],
+                        status: action,
+                    }
+                });
+            }
+            location.reload();
+        }
+
+        else if(action == "Change Revision"){
```

Comment:
when we update the revision there is a good change we will need to update the branch as well- a follow up PR/bug could be to make this more streamlined as well as updating the tbplurl generated link.